### PR TITLE
Add support for Asturian

### PR DESCRIPTION
**Pull request by softastur**
All steps to add a new layout, as described in https://github.com/florisboard/florisboard/blob/master/CONTRIBUTING.md, have been completed so Asturian layout should be ready to use.

### CODE REVIEW

File: app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/asturian.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[
```
Comment (patrickgold) [inline app/src/main/assets/ime/keyboard/org.florisboard.layouts/layouts/characters/asturian.json:1]: This file is identical to qwerty.json, and in fact in the default subtype you use QWERTY directly, so I suppose we can remove this json file as it is unused.

File: app/src/main/assets/ime/keyboard/org.florisboard.localization/extension.json [hunk 2]
```diff
@@ -644,6 +645,15 @@
...
+    },
+    {
+      "languageTag": "ast",
+      "composer": "org.florisboard.composers:appender",
+      "currencySet": "org.florisboard.currencysets:euro",
+      "popupMapping": "org.florisboard.localization:ast",
+      "preferred": {
+        "characters": "org.florisboard.layouts:qwerty"
+      }
+    },
```
Comment (patrickgold) [inline app/src/main/assets/ime/keyboard/org.florisboard.localization/extension.json:657]: The last comma after the curly brace is invalid JSON and causes a parse error, removing it fixes the problem.
 
```suggestion
    }
```

**Review by patrickgold — CHANGES_REQUESTED** [review]
Hey,

first of all I am very sorry for not reviewing your pull request for so long, last few months have been pretty stressful for me and thus I haven't really had much time for FlorisBoard, hope you can understand!

I've reviewed your changes and tested your layout on-device and it looks good, however there are two things that need to be changed (see attached review comments on the specifics). Once they are fixed I will happily merge in your changes and release them with the next alpha release :)

**Comment by softastur** [comment]
I've just screw things up and all changes were deleted when trying to sync repos. I will create a new PR in a few minutes containing all fixes and suggestions............

**Pull request closed without merging**